spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowServerConfiguration.java

```diff
@@ -162,12 +164,28 @@ public Object postProcessAfterInitialization(Object bean, String s) throws Beans
 
 	@Bean
 	@ConditionalOnMissingBean
+	@ConditionalOnExpression("#{'${serverStore:redis}'.equals('db')}")
+	public StreamDefinitionRepository rdbmsStreamDefinitionRepository(DataSource dataSource) {
+		return new RdbmsStreamDefinitionRepository(dataSource);
+	}
+
+	@Bean
+	@ConditionalOnExpression("#{'${serverStore:redis}'.equals('db')}")
+	public DeploymentIdRepository rdbmsDeploymentIdRepository(DataSource dataSource) {
+		return new RdbmsDeploymentIdRepository(dataSource);
+	}
+
+	@Bean
+	@ConditionalOnMissingBean
+	@ConditionalOnExpression("#{'${serverStore:redis}'.equals('redis')}")
```

Comment:
if we are keeping redis as a repository the method name should be redisStreamDefinitionRepository.